src/app/(dashboard)/my-agents/page.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+import AgentCard from '@/components/ui/agents/AgentCard';
+import { Button } from '@/components/ui/button';
+import { useMyAssistants } from '@/hooks/useAssistants';
+import { useBitteWallet } from '@bitte-ai/react';
+import Link from 'next/link';
+
+export default async function MyAgents() {
+  const { activeAccountId } = useBitteWallet();
+
+  const { agents, loading } = useMyAssistants(activeAccountId || '');
```

Comment:
this OR empty string trick is not a very appropriate way to handle things. Especially at such an early stage. Looking at the logic of the code in `useMyAssistants`, it seems as if this empty string is not even considered. As far as I can tell we lazily check `if (accountId)` so we are indirectly making use of trick. Would prefer to see the function accept `string | undefined` (or whatever the data type of `activeAccountId` is). Overall, we should not have to be manipulating values here.

```suggestion
  const { agents, loading } = useMyAssistants(activeAccountId);
```

Reply:
updated!